.github/workflows/arm-docker-latest-push.yml

```diff
@@ -0,0 +1,39 @@
+name: Build Latest ARM Docker Image
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [staging]
+
+permissions:
+  contents: read
+
+jobs:
+  publish_latest:
+    name: Publish latest tagged image
+    runs-on: ubuntu-latest
+    env:
+      PUSH_TO_DOCKER: ${{github.ref == 'refs/heads/staging'}}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Get version
+        run: echo "LOGFLARE_VERSION=$(cat VERSION)" >> $GITHUB_ENV
+      - name: Get commit SHA
+        run: echo "GITHUB_SHA=$(git rev-parse HEAD)" >> $GITHUB_ENV
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Build Version
+        uses: docker/build-push-action@v3
+        with:
+          push: ${{env.PUSH_TO_DOCKER}}
+          tags: supabase/logflare:latest
```

Comment:
usually `latest` is for the latest version image, lets use a different tag instead like `next` for both this workflow and the other workflow file for staging deploy.

Reply:
also, should add the arm suffix, so `next-arm`

Reply:
that is on purpose, if we have that we will need to have some docker compose weirdness depending on the platform of the user running it. 

Since we are not doing semver at the moment it's ok to override for now and later with dedicated machines and a more workable time to docker push we can move this into our main GH action flow.